crates/spirv-builder/src/lib.rs

```diff
@@ -204,7 +204,7 @@ fn invoke_rustc(builder: &SpirvBuilder, multimodule: bool) -> Result<PathBuf, Sp
         .unwrap_or_default();
 
     let rustflags = format!(
-        "-Z codegen-backend={} -Z symbol-mangling-version=v0{}",
+        "-Z codegen-backend={} {}",
```

Comment:
Wait, why was this done? I probably missed a discussion around this, but I couldn't find anything in the review comments here.

Reply:
So, it was required for a previous iteration of the PR, however my most recent changes removed the parts of `Image` that required the newer symbol mangling version, so this is no longer needed.